media/tracker_modules/xm.ksy

```diff
@@ -0,0 +1,280 @@
+meta:
+  id: xm
+  title: Extended Module
+  application:
+    - FastTracker 2
+    - Protracker
+    - MilkyTracker
+    - libmodplug
+    - Mikmod
+    - Extended Module Player
+    - VLC
+    - Media Player Classic
+    - MPlayer
+  file-extension: xm
+  license: Unlicense
+  endian: le
+  encoding: utf-8
+doc-ref: |
+  http://sid.ethz.ch/debian/milkytracker/milkytracker-0.90.85%2Bdfsg/resources/reference/xm-form.txt
+  ftp://ftp.modland.com/pub/documents/format_documentation/FastTracker%202%20v2.04%20(.xm).html
+  
+seq:
+  - id: preheader
+    type: preheader
+  - id: header
+    size: preheader.header_size - 4
```

Comment:
You don't like prefixes proposed in the style guide that much (i.e. `len_`, `num_`, `ofs_`, etc)?

Reply:
Yes, I really don't like them:
1 it is a bit weird to put low-level details first and high-level last
2 combining verbose words with non-verbose ones also looks weird

Reply:
Well, could you raise your concerns in https://github.com/kaitai-io/kaitai_struct/issues/140. Current "silent protest" won't get us anywhere — I still think that any standard at all is better than no standard, and until we'll discuss your objections, I would just take time to enforce it manually, i.e. by fixing your contributions. This will, in turn, will probably just annoy you, because:

* It takes longer time to get your ksys into this repository
* After that, "that guy" comes and mangles it in a way you don't like it

I see that you do a lot of interesting stuff here, and it would be bad if all that would go to waste just because we couldn't agree on some basic stuff.

I actually have an idea. Can I contact you using your mail.ru address?

Reply:
>I actually have an idea. Can I contact you using your mail.ru address?
Of course you can, but why not to discuss it on GH. Have I made your furious?

>I would just take time to enforce it manually, i.e. by fixing your contributions.

Completely OK.

Also
https://github.com/kaitai-io/kaitai_struct_doc/pull/7

Reply:
> If you contact by email, please leave a comment on GH about that.

Confirming, I've sent you the e-mail.